WallPaper_Project1/WallPaper_Project1/Data/Source/Remote/API/APICaller.swift

```diff
@@ -10,41 +10,50 @@ import Foundation
 class APICaller {
     static let shared = APICaller()
     private let session: URLSession
+
     private init() {
         let config = URLSessionConfiguration.default
         session = URLSession(configuration: config)
     }
+    let headers = [
+        "Authorization": "563492ad6f9170000100000116158da11e414c30a7a6352f814a235a"
+    ]
+
     func getJSON<T: Codable>(urlApi: String, completion: @escaping (T?, Error?) -> Void) {
         guard let url = URL(string: urlApi) else {
             return
         }
         var request = URLRequest(url: url)
+        request.allHTTPHeaderFields =  headers
```

Comment:
```suggestion
        request.allHTTPHeaderFields = headers
```
Check lại Swiftlint đi em @@ nhiều lỗi convention quá